test/branch.ts

```diff
@@ -48,19 +51,18 @@ describe('Branch module', async () => {
       // // tests
       const headSHA = await getBranchHead(logger, octokit, origin, 'master');
       expect(headSHA).to.equal(branchResponse.data.commit.sha);
-      sinon.assert.calledOnce(getBranchStub);
       sinon.assert.calledOnceWithExactly(getBranchStub, {
         owner: origin.owner,
         repo: origin.repo,
         branch: 'master',
       });
-
-      // restore
-      getBranchStub.restore();
     });
   });
 
   describe('The create branch function', () => {
```

Comment:
A bit of a nit, but in most cases I'll make the describe call out a specific Function or Class, vs., a phrase like in an `it` statement:

```
describe('FooClas', () => {
  describe('barMethod', () => {
     it("has some behavior I'm testing")
     it("exhibits some other behavior I'm testing")
  })
})
```